validator/src/main.rs

```diff
@@ -2564,6 +2579,21 @@ pub fn main() {
         None
     };
 
+    let accountsdb_plugin_config_file = if matches.is_present("enable_accountsdb_plugin") {
+        Some(PathBuf::from(
+            matches
+                .value_of("accountsdb_plugin_config")
+                .unwrap_or_else(|| {
+                    eprintln!(
+                        "The --accountsdb-plugin-config <plugin_config_file> is required when --enable-accountsdb-plugin is set.",
```

Comment:
`clap` can enforce this for us. https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.requires

Reply:
ok, will look into it

Reply:
Done